.gitignore

```diff
@@ -1 +1,4 @@
 /target
+/dummy_data
+/scripts
+/.Scratch
```

Comment:
I ran the test and didn't get any of these folders created, what are they additional ignores for? Are they needed?

It's also worth noting that the commit that adds them should explain what they are for fc930ee4cf12ca633e2a3a75ad0d5a25535fc519